trio/_core/tests/test_run.py

```diff
@@ -1590,6 +1591,15 @@ def generator_based_coro():  # pragma: no cover
                 bad_call(len, [1, 2, 3])
             assert "appears to be synchronous" in str(excinfo.value)
 
+            @async_generator
+            async def async_gen(arg):  # pragma: no cover
+                yield_(await sleep(arg))
```

Comment:
I guess using the `pass` statement would make it clear that this line is never executed?